sdk/eventhub/Azure.Messaging.EventHubs/src/TrackOneClient/Amqp/AmqpEventDataSender.cs

```diff
@@ -100,6 +101,18 @@ protected override async Task OnSendAsync(IEnumerable<EventData> eventDatas, str
             } while (shouldRetry);
         }
 
+        internal override async ValueTask EnsureLinkAsync()
+        {
+            if (this.linkCreated)
+            {
+                return;
+            }
+
+            await this.SendLinkManager.GetOrCreateAsync(TimeSpan.FromSeconds(1)).ConfigureAwait(false);
```

Comment:
Still playing with this.    For some reason, without the delay, it blocks the link and causes publishing to timeout.    The AMQP library isn't well documented, so there's a good chance that I'm missing a step in marking my operation complete.

Reply:
I can't seem to figure out exactly why, but without the delay, something ends up locked in a way that causes timeouts for subsequent operations.   Even a 1ms delay seems to be effective, but a 0ms delay exhibits the timeout issues.

Going to accept this as a consequence of the track one implementation, for the time being.   My hope is that this will no longer be a thing when we complete the move of the AMQP layer to track two.